app/models/miq_worker.rb

```diff
@@ -299,6 +299,14 @@ def self.log_status_all(level = :info)
     MiqWorker.log_status(level)
   end
 
+  def self.cve_check
+    # TODO: optimize by keeping last hashes of cve streams raisng only upon update
```

Comment:
typo:  raisng

Why is this in `MiqWorker`?  Maybe `ManageIQ::Providers::ContainerManager` is a better place for it?